vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -60,6 +60,20 @@ class ViewShadowNode : public ShadowNodeBase {
         });
   }
 
+  void dispatchCommand(const std::string &commandId, const folly::dynamic &commandArgs) override {
+    if (commandId == "focus") {
+      if (auto uiManager = GetViewManager()->GetReactContext().NativeUIManager()) {
```

Comment:
do either of these need to get null checked?

Reply:
I followed the convention after the recent refactor by @acoates-ms - TextInputViewManager doesn't do the null check, so I assume that's fine?